Tomcat tests fail with NoClassDefFoundError: org/testcontainers/containers/ContainerFetchException

### Describe the bug

```
(1) [ERROR] ClassSelector [className = 'org.infinispan.test.integration.thirdparty.remote.HotRodIT', classLoader = ParallelWebappClassLoader
  context: infinispan-server-integration
  delegate: false
----------> Parent Classloader:
java.net.URLClassLoader@1f7030a6
] resolution failed
    Source: ClassSource [className = 'org.infinispan.test.integration.thirdparty.remote.HotRodIT', filePosition = null]
            at org.infinispan.test.integration.thirdparty.remote.HotRodIT.<no-method>(SourceFile:0)
    Cause: java.lang.NoClassDefFoundError: org/testcontainers/containers/ContainerFetchException

```

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

--- a/integrationtests/appserver/src/test/java/org/infinispan/test/integration/thirdparty/remote/HotRodIT.java
+++ b/integrationtests/appserver/src/test/java/org/infinispan/test/integration/thirdparty/remote/HotRodIT.java
@@ -31,7 +31,6 @@ import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.Assumptions;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
-import org.testcontainers.containers.ContainerFetchException;
 
 /**
  * Tests the Infinispan Hot Rod client integration with third-party application servers.
@@ -51,8 +50,11 @@ public class HotRodIT {
          InfinispanTestServer server = new InfinispanTestServer();
          server.start();
          SERVER = server;
-      } catch (ContainerFetchException e) {
-         Assumptions.abort("Container image not available: " + e.getMessage());
+      } catch (Exception e) {
+         if (e.getClass().getName().equals("org.testcontainers.containers.ContainerFetchException")) {
+            Assumptions.abort("Container image not available: " + e.getMessage());
+         }
+         throw e;
       }
    }
 

